logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -234,6 +235,13 @@ internal class UserConfigDataSource internal constructor(
             }
         }
 
+    override suspend fun clearE2EISettings() {
+        wrapStorageRequest {
+            userConfigStorage.setE2EISettings(null)
+            userConfigStorage.setE2EINotificationTime(0)
+        }
+    }
+
```

Comment:
do we need to to first check if the user have E2EI or it is safe to call those function anyway ?

Reply:
in fact it's just a records in SharedPrefs, so quit safe to call it :)